packages/wasm/crate/src/metadata.rs

```diff
@@ -61,3 +66,117 @@ fn shorten_id(captures: &regex::Captures) -> WasmResult<String> {
         .take(SHORTENED_ID_LENGTH)
         .collect())
 }
+
+#[cfg(test)]
+mod test_helpers {
+    use penumbra_proto::core::asset::v1::DenomUnit;
+
+    use super::*;
+
+    pub fn get_metadata_for(display_denom: &str) -> Metadata {
+        let mut denom_units = Vec::new();
+        denom_units.push(DenomUnit {
+            aliases: Vec::new(),
+            denom: format!("u{display_denom}"),
+            exponent: 0,
+        });
+        denom_units.push(DenomUnit {
+            aliases: Vec::new(),
+            denom: String::from(display_denom),
+            exponent: 6,
+        });
+
+        Metadata {
+            base: format!("u{display_denom}"),
+            description: String::from(""),
+            denom_units,
+            display: String::from(display_denom),
+            images: Vec::new(),
+            name: String::from(""),
+            penumbra_asset_id: None,
+            symbol: String::from(""),
+        }
+    }
+
+    #[test]
+    fn it_interpolates_display_denom() {
```

Comment:
This is a test for a test helper, which... might be overkill 😆 but I was taking it as an opportunity to practice my newly acquired Rust testing skills.